src/app/components/Form/PasswordInput/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
```

Comment:
> Hmm instead of adding an extra PasswordInput component, can't we make the Textfield more flexible to support adding an icon?

@dylancom :D had another call in between writing and posting this. Agreed!